src/core/thread/mesh_forwarder.cpp

```diff
@@ -2210,10 +2215,21 @@ void MeshForwarder::LogIp6Message(MessageAction aAction, const Message &aMessage
     {
     case kMessageReceive:
         actionText = "Received";
+        mIpCounters.mRxSuccess++;
         break;
 
     case kMessageTransmit:
-        actionText = (aError == OT_ERROR_NONE) ? "Sent" : "Failed to send";
+        if (aError == OT_ERROR_NONE)
+        {
+            actionText = "Sent";
+            mIpCounters.mTxSuccess++;
```

Comment:
These counters will only increment if both of the following are true:
- `OPENTHREAD_CONFIG_LOG_LEVEL >= OT_LOG_LEVEL_INFO`
- `OPENTHREAD_CONFIG_LOG_MAC == 1`

Are we okay with that?